### ISSUE TITLE
Need more details about id format used

### ISSUE THREAD

**Issue by yannroux** [microsoftgraph/microsoft-graph-docs#3463]
If the special header Prefer: IdType="ImmutableId" is added to the request, what is the format used ? restImmutableEntryId or immutableEntryId ? or another one ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 77dc8516-103a-7df2-2685-1ce9045941b5
* Version Independent ID: 90ebf4ab-1563-4a6d-d184-c91159352366
* Content: [Get immutable identifiers for Outlook resources](https://docs.microsoft.com/en-us/graph/outlook-immutable-id)
* Content Source: [concepts/outlook-immutable-id.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/live/concepts/outlook-immutable-id.md)
* Service: **microsoft-graph**
* GitHub Login: @jthake-msft
* Microsoft Alias: **jthake-msft**

**Comment by jasonjoh** [comment]
It's the `restImmutableEntryId` format, since Graph is a REST API. This is documented here: https://docs.microsoft.com/en-us/graph/api/user-translateexchangeids?view=graph-rest-beta

**Comment by jthake-msft** [comment]
@jasonjoh  how do they get to that page? Do we reference it in a parent page somewhere? It's not currently in the TOC.

**Comment by jasonjoh** [comment]
Yeah it's not in the TOC because it's sort of a niche API. It's linked on the page @yannroux is referring to here: https://docs.microsoft.com/en-us/graph/outlook-immutable-id#updating-existing-data

It's also linked in the method table on the user resource page (since the API is off of the user object): https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-beta
